ansible/fluffy.yml

```diff
@@ -27,3 +27,13 @@
       with_sequence: start=1 end=32
       loop_control:
         loop_var: index
+
+- name: Configure Nimbus Portal Bridge
+  become: true
+  vars_files: portal-bridge.yml
+  hosts:
+    - metal-01.ih-eu-mda1.nimbus.fluffy
+  roles:
+    - { role: infra-role-open-ports,    tags: open-ports    }
```

Comment:
This is incorrect. Open ports role should be run just once.

Reply:
> This is incorrect. Open ports role should be run just once.

It is incorrect if we would have open_ports defined just in the group_vars file, but we are defining addtional rule in the portal-bridge.yml since it is needed only on the host `metal-01.ih-eu-mda1.nimbus.fluffy`. I mean I can move it in the group vars file but then we need to be ok with the fact additional port will be open to public without a reason.

Reply:
Oh I see, because there's two hosts. I forgot we added a second fluffy host.

I guess that's fine, though I should consider how we could make `open-ports` rules optional. Should be quite easy.